tests/load/sources/filesystem/test_filesystem_source.py

```diff
@@ -96,6 +96,7 @@ def assert_csv_file(item: FileItem):
     assert len(list(nested_file | assert_csv_file)) == 1
 
 
+@pytest.mark.skip("Needs secrets toml to work..")
```

Comment:
maybe we should have a place for source tests where secrets are present. we can probably fix this with the right ENV var though